test/config_parser_SUITE.erl

```diff
@@ -56,10 +55,24 @@ s2s(Config) ->
     TOML_path = ejabberd_helper:data(Config, "s2s_only.toml"),
     State2 = mongoose_config_parser_toml:parse_file(TOML_path),
     Opts2 = mongoose_config_parser:state_to_opts(State2),
-        
+
     compare_unordered_lists(lists:filter(fun filter_config/1, Opts1), Opts2,
                             fun handle_config_option/2).
 
+modules(Config) ->
```

Comment:
I think we could extract the common parts of these tests to a helper function, as they are getting repetitive. I think we could separate the equivalence tests to a different suite as well. However, this can be done in a separate PR.